.travis.yml

```diff
@@ -1,17 +1,62 @@
 language: python
 dist: xenial
 sudo: true
-matrix:
-  include:
-    - python: 3.6
-    - python: 3.7
+python:
+  - 3.6
+  - 3.7
 before_install:
   - sudo add-apt-repository ppa:maarten-fonville/protobuf -y
   - sudo apt-get update -q
   - sudo apt-get install protobuf-compiler
 install:
-  - pip install tox-travis codecov
+  - pip install tox-travis codecov wheel
 script:
   - tox
 env:
   - 'COVERAGE_COMMAND=codecov -e TOXENV'
+
+jobs:
+  include:
+    - stage: archive
+      script:
+        - python setup.py bdist_wheel
+        - |
```

Comment:
Please add a comment here explaining that the parentheses in `(.*\\\\.whl)` are not used as a capture group and cannot be removed.